backend/wmg/data/schemas/cube_schema.py

```diff
@@ -42,13 +71,26 @@
     ]
 )
 
+expression_summary_fmg_domain = tiledb.Domain(
+    [
+        tiledb.Dim(name=cube_indexed_dim, domain=None, tile=None, dtype="ascii", filters=filters)
+        for cube_indexed_dim in expression_summary_fmg_indexed_dims
+    ]
+)
+
 # The cube attributes that comprise the core data stored within the cube.
 expression_summary_logical_attrs = [
-    tiledb.Attr(name="n_cells", dtype=np.uint32, filters=filters),
```

Comment:
note: call this change out in the PR, since it's modifying the existing WMG expr summary cube schema